File: testes/testComp/3.casos_teste_semantico_gerador/saida/cobertura_gerador/typescript/controllers/user-profiles.controller.ts
import { Controller, Get, Put, Patch, Delete } from '@nestjs/common';
import { UserDto } from '../dtos/user.dto';

@Controller()
export class UserProfilesController {

  @Get('/user-profiles')
  getUserProfiles(): UserDto[] {
    throw new Error('Not implemented');
  }

  @Get('/user-profiles/:id')
  getUserProfilesById(): UserDto {
    throw new Error('Not implemented');
  }

  @Put('/user-profiles/:id')
  updateUserProfileById(): UserDto {
    throw new Error('Not implemented');
  }

  @Patch('/user-profiles/:id/active')
  patchUserProfileByIdActive(): boolean {
    throw new Error('Not implemented');
  }

  @Delete('/user-profiles/:id')
  deleteUserProfileById(): boolean {
    throw new Error('Not implemented');
  }
}

File: testes/testComp/3.casos_teste_semantico_gerador/saida/cobertura_gerador/kotlin/controllers/UserProfilesController.kt
package com.example.generated.controllers

import com.example.generated.dtos.UserDto
import org.springframework.web.bind.annotation.*

@RestController
@RequestMapping
class UserProfilesController {

    @GetMapping("/user-profiles")
    fun getUserProfiles(): List<UserDto> {
        TODO("Not yet implemented")
    }

    @GetMapping("/user-profiles/{id}")
    fun getUserProfilesById(): UserDto {
        TODO("Not yet implemented")
    }

    @PutMapping("/user-profiles/{id}")
    fun updateUserProfileById(): UserDto {
        TODO("Not yet implemented")
    }

    @PatchMapping("/user-profiles/{id}/active")
    fun patchUserProfileByIdActive(): Boolean {
        TODO("Not yet implemented")
    }

    @DeleteMapping("/user-profiles/{id}")
    fun deleteUserProfileById(): Boolean {
        TODO("Not yet implemented")
    }
}
